README.md

```diff
@@ -1,29 +1,48 @@
 [![Promote charm](https://github.com/canonical/chrony-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/chrony-operator/actions/workflows/promote_charm.yaml)
 [![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 
-A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) 
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
 for deploying and managing the [Chrony](https://chrony-project.org) NTP server 
 in your systems.
 
 This charm simplifies the configuration and maintenance of `chrony` across a 
 range of environments, enabling basic time provision and synchronization with 
 NTP servers.
 
-## Project and community
+## Get started
+In this section, we will deploy the Chrony charm, which can be used as a time source for NTP clients.  
+You’ll need a workstation, e.g., a laptop, with sufficient resources to launch a virtual machine with 4 CPUs, 8 GB RAM, and 50 GB disk space.
+
+### Set up
+You can follow the tutorial [here](https://juju.is/docs/juju/set-up--tear-down-your-test-environment#heading--set-up-automatically) to set up a test environment for Juju with LXD.
+
+### Deploy
+Deploy the Chrony charm using the `juju deploy` command.
 
-The Chrony Operator is a member of the Ubuntu family. It's an
-open source project that warmly welcomes community projects, contributions,
-suggestions, fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-* [Get support](https://discourse.charmhub.io/)
-* [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
-* [Contribute](https://charmhub.io/chrony/docs/contributing)
-* [Roadmap](https://charmhub.io/chrony/docs/roadmap)
-Thinking about using the Chrony charm for your next project? [Get in touch](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)!
+```
+ubuntu@tutorial-vm:~$ juju deploy chrony --channel latest/edge
+Deployed "chrony" from charm-hub charm "chrony", revision 42 in channel latest/edge on ubuntu@24.04/stable
+```
 
----
+### Basic operations
+When the Chrony charm has finished deployment and installation, it should show the message `no time source configured`.  
+Now we can run the `juju config` command to configure an upstream time source for the Chrony charm. In this example, we will use Ubuntu's time server.
 
-For further details,
-[see the charm's detailed documentation](https://charmhub.io/chrony/docs).
+```
+ubuntu@tutorial-vm:~$ juju config chrony sources=ntp://ntp.ubuntu.com
+```
+
+Shortly after the configuration, the Chrony charm should transition into the active state and be able to provide time to other NTP clients.
+
+## Learn more
+* [Read more](https://charmhub.io/chrony)
+* [Troubleshooting](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
+
```

Comment:
Is there an official website we could include in this list, perhaps https://chrony-project.org/?